docs/user/ovirt/install_upi.md

```diff
@@ -0,0 +1,600 @@
+ignition config files.
+One of the recommended ways is to use a DHCP server to manage the machines in the long-term, maybe configuring the DHCP server
+itself to provide persistent IP addresses and host names to the cluster machines.
+
+Network connectivity between machines should be configured to allow cluster components to communicate:
+
+- Kubernetes NodePort
+  Machines require connectivity to every other machine for OpenShift platform components through the port range `30000`-`32767` .
+
+- OpenShift reserved
+  Connectivity to reserved port ranges `10250`-`10259` and `9000`-`9999` should be granted on every machine.
+
+- Machines to control-plane
+  Connectivity to ports on ranges `2379`-`2380` (for etcd, peer and metrics) is required for control-plane machines and on
+  port `6443` for Kubernetes API.
+
+
+### Load Balancers
+Before installing the OpenShift Container Platform, two load balancers (layer-4) must be provided by the user infrastructure,
+one for the API and one for the Ingress Controller (to allow ingress to applications).
+
+- Load balancer for port `6443` and `22623` on control-plane and bootstrap machines (the bootstrap can be removed after control-plane
+  initialization completes).
+  The `6443` must be both internal and external reachable and is needed by the Kubernetes API server.
+  Port `22623` must be accessible to nodes within the cluster.
+
+- Load balancer for port `443` and `80` for machines running the ingress router (usually worker nodes in the default configuration).
+  Both ports must be accessible from within and outside the cluster.
+
+
+### DNS
+The UPI installation process requires the user to setup the existing infrastructure provided DNS to allow the correct resolution of
+the main components and services
+
+- Kubernetes API
+  DNS records `api.<cluster_name>.<base_domain>` (internal and external resolution) and `api-int.<cluster_name>.<base_domain>` 
+  (internal resolution) must be added to point to the Load balancer targeting the control plane machines. 
+
+- etcd
+  For each control plane machine (by default only master nodes, excluding the bootstrap), DNS records 
+  `etcd-<idx>.<cluster_name>.<base_domain>` must be added to point to `<idx>`'th control plane machine (internal resolution).
+
+  For each control plane machine a SRV type DNS record must be provided, like:
+
+    ```plain
+    # _service._proto.name.                            TTL   class SRV priority weight port target.
+    _etcd-server-ssl._tcp.$cluster_name.$base_domain   86400 IN    SRV 0        10     2380 etcd-0.$cluster_name.$base_domain.
+    _etcd-server-ssl._tcp.$cluster_name.$base_domain   86400 IN    SRV 0        10     2380 etcd-1.$cluster_name.$base_domain.
+    _etcd-server-ssl._tcp.$cluster_name.$base_domain   86400 IN    SRV 0        10     2380 etcd-2.$cluster_name.$base_domain.
+    ```
```

Comment:
This should no longer be necessary, can you test with out this and remove this?

Reply:
Thx @sdodson, tested without and working.
Updated the documentation